backend/automatic_walk_time_tables/utils/path.py

```diff
@@ -5,7 +5,7 @@
 import polyline
 
 from .point import Point_LV03, Point
-from .way_point import WayPoint
+from .way_point import WayPoint, way_point_from_json
```

Comment:
We should consider using package imports instead of relative imports (see other files)